src/building/build-install-distribution-artifacts.md

```diff
@@ -7,23 +7,25 @@ You’ll want to run this command to do it:
 ./x dist
 ```
 
-# Install distribution artifacts
+# Install from source
 
-If you’ve built a distribution artifact you might want to install it and
-test that it works on your target system. You’ll want to run this command:
+You might want to prefer installing Rust(and tools configured in your configuration)
+by building from source. If so, you want to run this command:
 
 ```bash
 ./x install
 ```
 
-   Note: If you are testing out a modification to a compiler, you
-   might want to use it to compile some project.
-   Usually, you do not want to use `./x install` for testing.
-   Rather, you should create a toolchain as discussed in
-   [here][create-rustup-toolchain].
+   Note: If you are testing out a modification to a compiler, you might
+   want to build the compiler (with `./x build`) then create a toolchain as
```

Comment:
Nit: probably `./x build library --stage 1` just to be explicit here.

Reply:
Default stage is already `1` and `build library` doesn't really make it clearer (as we say 'build the compiler'). I prefer to show the shortest and most straightforward commands whenever possible while relying on bootstrap defaults.

Reply:
Fair enough.